.gitignore

```diff
@@ -7,3 +7,4 @@ node_modules/
 local.properties
 *.bundle
 *.zip
+.DS_Store
```

Comment:
This is best left to your global `.gitignore` file to keep out non-project specific ignores. You can use mine as a reference: https://github.com/kasperisager/dotfiles/blob/main/home/.gitignore

Reply:
I did not even now we can globalize .gitignore haha! Thanks